cmd/kubeadm/app/constants/constants.go

```diff
@@ -500,6 +500,7 @@ var (
 		21: "3.4.13-0",
 		22: "3.5.1-0",
 		23: "3.5.1-0",
+		24: "3.5.3-0",
```

Comment:
we should also change this https://github.com/kubernetes/kubernetes/blob/654b0c6a7e1be21b7345e9fb0fadb8637747e230/cmd/kubeadm/app/constants/constants.go#L323 , @justaugustus 

but i don't think kubeadm needs more changes.

Reply:
Sounds good. thanks for looking @neolit123